### DISCUSSION TITLE
Migrate SQL DB with its Data

### DISCUSSION THREAD

**Discussion by UmerTariqKhan** [mixcore/mix.core#448] [Q&A] [answered]
I pulled new code from the development branch see lots of changes in code and DB, Now problem is to migrate the old DB with its data. I tried but (1) gives an error (2). Don't know which table needs to be recreated. Please help and mention the steps. Thanks Alot 

1- "ctx.Database.Migrate();" 
2- "Change the IDENTITY property of a column, the column needs to be dropped and recreated".

![image](https://user-images.githubusercontent.com/36357077/127924778-2611a3c3-0ebc-4b4e-bbf9-957940e2cf40.png)

**Comment by nhathoang989** [comment]
Hi UmerTariqKhan,
If you can supply the prior version you used, I'll be able to assist you in resolving this issue.

**Comment by UmerTariqKhan** [comment]
Thanks for your support.

Here Test version is deployed
http://54.159.73.158:8081/

**Comment by UmerTariqKhan** [comment]
username : super-admin
password : Pass88550

**Comment by UmerTariqKhan** [comment]
here is my db
https://drive.google.com/file/d/1XPCXkGPXLf82h5mLkTVBB43uC1-Svq0Z/view?usp=sharing

**Comment by UmerTariqKhan** [comment]
My current mix core version is 1.0.1

**Reply by nhathoang989** [reply]
I'll check over your database.
By the way, I was just trying to update from v1.0.1 and came across these remarks.
https://github.com/mixcore/mix.core/discussions/450

**Comment by nhathoang989** [comment, marked as answer]
> I pulled new code from the development branch see lots of changes in code and DB, Now problem is to migrate the old DB with its data. I tried but (1) gives an error (2). Don't know which table needs to be recreated. Please help and mention the steps. Thanks Alot
> 
> 1- "ctx.Database.Migrate();"
> 2- "Change the IDENTITY property of a column, the column needs to be dropped and recreated".
> 
> ![image](https://user-images.githubusercontent.com/36357077/127924778-2611a3c3-0ebc-4b4e-bbf9-957940e2cf40.png)

I've recently updated and merged to master due to a problem in SqlServer Migration. Please re-pull the newest source from master. 
https://github.com/mixcore/mix.core/pull/451/commits/7ed55acf10c110bc4b1ae8c9d73da980c5d3e14c

**Comment by UmerTariqKhan** [comment]
Ok let me try

**Comment by UmerTariqKhan** [comment]
Thanks soo much
